plugins/essif-lab/src/Integrations/WordPressDataHandler.php

```diff
@@ -40,17 +40,26 @@ private function handleModelFieldData(Model $model, array $data): array {
 		$old = json_decode($description, true);
 		$new = is_array($old) ? $old : [];
 
-		return $this->handleModelFieldDataSignature($data, $new);
+        if (array_key_exists(Constants::FIELD_TYPE_SIGNATURE, $data)) {
+            $new = $this->handleModelFieldDataSignature($data, $new);
+        }
+
+        $immutable = array_key_exists(Constants::FIELD_TYPE_IMMUTABLE, $data) ? true : false;
+        $this->handleModelFieldDataImmutable($model, $immutable);
+
+		return $new;
 	}
 
 	private function handleModelFieldDataSignature(array $data, array $new): array {
-		if (array_key_exists(Constants::FIELD_TYPE_SIGNATURE, $data)) {
```

Comment:
Moest deze echt naar `handelModelFieldData`?

Reply:
ik dacht dat het netter was, want als er geen field signature is, wil je niet echt een functie aanroepen die handleModelFieldDataSignature heet, want er is niets om te handlen